pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -132,7 +141,12 @@ let
         makeWrapper ${lib.escapeShellArgs finalMakeWrapperArgs} ${wrapperArgsStr}
       '';
 
-    paths = [ neovim ];
+    buildPhase = ''
+      mkdir -p $out
+      for i in ${neovim}; do
+        ${lndir}/bin/lndir -silent $i $out
```

Comment:
```suggestion
        ${lib.getExe buildPackages.lndir} -silent $i $out
```

Reply:
without using buildPackages or nativeBuildInputs lndir will be for host not for build

trivial-builders inherits lndir from buildPackages https://github.com/NixOS/nixpkgs/blob/9e1c5fd098a2a663e3279e42af96fd7235499bc9/pkgs/top-level/stage.nix#L108 so it's not visible in the file itself

Reply:
I am eager to complete this as this will unblock a bunch of features/makes them easier. I really dont wanna break cross build. Problem is I can't find any cross compile command that succeeds on master/unstable: `nix-build -A pkgsCross.aarch64-android.neovim` fails,   `nix build nixpkgs#legacyPackages.x86_64-linux.pkgsCross.aarch64-multiplatform.neovim` fails too. Any recommendation ?